generators/client-2/templates/src/main/webapp/app/account/social/_social-auth.component.ts

```diff
@@ -0,0 +1,27 @@
+import { Component, OnInit, Inject } from '@angular/core';
+import { StateService } from 'ui-router-ng2';
+import { AuthService } from '../../shared';
+import { LoginService } from '../../shared';
+
+@Component({
+    selector: '<%=jhiPrefix%>-register',
+    templateUrl: 'app/account/register/register.html'
+})
+export class SocialAuthComponent implements OnInit {
```

Comment:
rename to `SocialRegisterComponent` as thats what it does. You can keep the file name different based on authentication if needed